addons/report/tests/test_reports.py

```diff
@@ -2,35 +2,33 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 import logging
 
-import openerp
-import openerp.tests
+import odoo
+import odoo.tests
 
 
 _logger = logging.getLogger(__name__)
 
 
-@openerp.tests.common.at_install(False)
-@openerp.tests.common.post_install(True)
-class TestReports(openerp.tests.TransactionCase):
+@odoo.tests.common.at_install(False)
+@odoo.tests.common.post_install(True)
+class TestReports(odoo.tests.TransactionCase):
     def test_reports(self):
-        registry, cr, uid = self.registry, self.cr, self.uid
-        r_model = registry('ir.actions.report.xml')
         domain = [('report_type', 'like', 'qweb')]
-        for r in r_model.browse(cr, uid, r_model.search(cr, uid, domain)):
-            report_model = 'report.%s' % r.report_name
+        for report in self.env['ir.actions.report.xml'].search(domain):
+            report_model = 'report.%s' % report.report_name
             try:
-                registry(report_model)
+                self.env[report_model]
             except KeyError:
             # Only test the generic reports here
-                _logger.info("testing report %s", r.report_name)
-                report_model = registry(r.model)
-                report_model_ids = report_model.search(cr, uid, [], limit=10)
+                _logger.info("testing report %s", report.report_name)
+                report_model = self.env[report.model]
+                report_model_ids = report_model.search([], limit=10)
```

Comment:
change variable name.. search return recordset(s) not ids..
report_model_ids ==> report_models